cli/src/virtual_appliance/bootstrap.sh

```diff
@@ -20,7 +20,7 @@
 set -euE
 
 export LOG_FILE="${LOG_FILE:-/var/log/seagate/provisioner/va_bootstrap.log}"
-mkdir -p $(dirname ""${LOG_FILE}")"
+mkdir -p $(dirname "${LOG_FILE}")"
```

Comment:
The trailing `"`?
That needs to be removed as well.

Reply:
Oh ! I had missed that.
Done.